src/libraries/System.Private.CoreLib/src/System/Globalization/OrdinalCasing.Icu.cs

```diff
@@ -427,17 +439,33 @@ internal static unsafe int LastIndexOf(ReadOnlySpan<char> source, ReadOnlySpan<c
             }
         }
 
+        private static void InitCasingTable()
+        {
+            ushort[]?[] table = new ushort[]?[s_casingTableInit.Length * 8];
+            for (int i = 0; i < s_casingTableInit.Length * 8; ++i)
```

Comment:
Couldn't we start i at 16? The first 2 table init entries are `0b00000000`, I wonder if that can change? @tarekgh @GrabYourPitchforks ?

Reply:
TBH not sure if I'd optimize that away. True, iterating over everything does a little more work than needed, but we're only talking a small handful of unnecessary iterations, and this is a bit more future-proof. It's less likely to incur bugs should we ever need to change the early values.

Reply:
Yep, that's what I wanted to know if these could actually change in the future, if it can or we're not sure, it makes sense to make it future-proof and iterate over everything.